platerecognizer.js

```diff
@@ -0,0 +1,33 @@
+// adapted from https://docs.platerecognizer.com/?javascript#read-number-plates-from-an-image
+
+const fetch = require("node-fetch");
+const FormData = require("form-data");
+const fs = require("fs");
+
+require('dotenv').config();
+
+const { PLATERECOGNIZER_TOKEN } = process.env;
+
+let image_path = __dirname + "/PXL_20220617_203454473_50percent.jpg";
```

Comment:
plate recognizer has a file size limit of 3MB, and recommends an image resolution of 1024x768, so we'll probably need to use e.g. `sharp` to resize images to make them under 3MB: https://app.platerecognizer.com/upload-limit/

![image](https://user-images.githubusercontent.com/6473925/176568744-d40ffe9f-7444-4fe3-9592-47eecfc2f5a3.png)